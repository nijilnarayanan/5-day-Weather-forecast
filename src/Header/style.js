import styled from 'styled-components'

export const Wrapper = styled.section`
    display: flex;
    flex-direction:column;
    margin: 16px;
    border-radius: 8px;
    background-color: #e5e5e5;
    padding: 20px;

    @media screen and (min-width: 767px) {
        padding: 40px;
        margin: 32px;
    }
`
export const HeaderText = styled.div`
    font-size: 20px;
    text-align: center;
    margin-bottom: 10px;
    @media screen and (min-width: 767px) {
        font-size: 40px;
    }
`

export const Location = styled.div`
    display:flex;
    align-items: center;
    margin: 0 auto;
`

export const Icon = styled.img`
    height: 18px;
    margin-right: 10px;
    @media screen and (min-width: 767px) {
        height: 26px;
    }
`
export const City = styled.span`
    font-size : 18px;
    font-weight: 500;
    @media screen and (min-width: 767px) {
        font-size: 30px;
    }
`