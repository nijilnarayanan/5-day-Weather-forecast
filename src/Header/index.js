import React, { Fragment } from 'react'
import { Wrapper, HeaderText, Location, Icon, City } from './style'
import location from '../location.png'

const Header = () => {
	return (
		<Fragment>
			<Wrapper>
				<HeaderText>5 Day Weather Forecast</HeaderText>
				<Location>
					<Icon src={location} />
					<City>Mai</City>
				</Location>
			</Wrapper>
		</Fragment>
	)
}

export default Header
