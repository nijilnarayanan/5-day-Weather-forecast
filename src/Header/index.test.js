import React from 'react'
import { mount } from 'enzyme'
import Header from './index'

test('Run Test <Header />', () => {
	const component = mount(<Header />)
	expect(component.debug()).toMatchSnapshot()
})
