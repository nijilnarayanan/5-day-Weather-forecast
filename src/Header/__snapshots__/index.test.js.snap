// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Run Test <Header /> 1`] = `
"<Header>
  <styled.section>
    <section className=\\"sc-bdfBwQ jEWXLh\\">
      <styled.div>
        <div className=\\"sc-gsTCUz jLFiED\\">
          5 Day Weather Forecast
        </div>
      </styled.div>
      <styled.div>
        <div className=\\"sc-dlfnbm fGPqXe\\">
          <styled.img src=\\"location.png\\">
            <img src=\\"location.png\\" className=\\"sc-hKgILt gqiLmC\\" />
          </styled.img>
          <styled.span>
            <span className=\\"sc-eCssSg ldyRwL\\">
              Mai
            </span>
          </styled.span>
        </div>
      </styled.div>
    </section>
  </styled.section>
</Header>"
`;
